core/css.js

```diff
@@ -142,12 +142,13 @@ Blockly.Css.CONTENT = [
   '  background-color: #fff;',
   '  outline: none;',
   '  overflow: hidden;',  /* IE overflows by default. */
+  '  display: block;', 
   '}',
 
   '.blocklyWidgetDiv {',
   '  display: none;',
   '  position: absolute;',
-  '  z-index: 999;',
+  '  z-index: 99999 !important;',
```

Comment:
2015-09-22 13:29 GMT+02:00 Carlos <notifications@github.com>:

> In core/css.js
> <https://github.com/google/blockly/pull/181#discussion_r40075434>:
>
> >    '}',
> >
> >    '.blocklyWidgetDiv {',
> >    '  display: none;',
> >    '  position: absolute;',
> > -  '  z-index: 999;',
> > +  '  z-index: 99999 !important;',
>
> But will it work without it?
>
If so, I'd recommend to remove it as you never know what the requirements
> of the developer might be or how it might interact with other frameworks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/blockly/pull/181/files#r40075434>.
>

You're right.
Without it, the div is still displayed. The real fix is to set an higher
z-index.